### PR TITLE
add functionality to create new table at the front-end

### PR DESCRIPTION
**Pull request by Zuretto**

### CODE REVIEW

File: frontend/src/lib/components/CreateTable.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    import { goto } from '$app/navigation';
+    import type {TableResponse} from "./CreateTable.models";
+
+    let username = '';
+
+    const createBoard = (): void => {
+        if (!username || !username.trim().length) {
+            return;
+        }
+        fetch(`http://localhost:8080/poker_api/v1/table`, {
```
Comment (kezc) [inline frontend/src/lib/components/CreateTable.svelte:11]: suggestion: extract it to a separate file . . .

**Review by kezc — APPROVED** [review]
lgtm

File: frontend/src/lib/components/CreateTable.svelte (hunk 1 above)
Comment (kezc) [inline frontend/src/lib/components/CreateTable.svelte:11]: thought: I know nothing about front end but I would love to separate UI and business logic

File: frontend/src/lib/components/CreateTable.spec.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import {describe, it, expect, beforeEach, vi} from 'vitest';
```
Comment (kezc) [inline frontend/src/lib/components/CreateTable.spec.ts:1]: nitpick: sometimes you make spaces after `{` and sometimes not....

File: frontend/src/lib/components/CreateTable.svelte (hunk 1 above)
Comment (KubaZm) [reply frontend/src/lib/components/CreateTable.svelte:11]: 🙂

File: backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/TableResource.kt [hunk 3]
```diff
@@ -2,12 +2,14 @@ package pl.poznan.put.tsd.planningpoker.backend.resources
...
 import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.CrossOrigin
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RestController
 import pl.poznan.put.tsd.planningpoker.backend.resources.responses.TableCreatedResponse
 import pl.poznan.put.tsd.planningpoker.backend.services.GamesService
 
+@CrossOrigin
```
Comment (KubaZm) [inline backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/TableResource.kt:12]: Isn't it possible to enable CORS globally in application.properties file?

**Comment by Zuretto** [comment]
created new PR: https://github.com/Zuretto/planning-poker/pull/7

File: backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/TableResource.kt (hunk 3 above)
Comment (Zuretto) [reply backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/TableResource.kt:12]: I deleted it in new PR

File: frontend/src/lib/components/CreateTable.svelte (hunk 1 above)
Comment (Zuretto) [reply frontend/src/lib/components/CreateTable.svelte:11]: I put the call in new file; for the logic, I think putting the code inside <script> tag is enough

Comment (Zuretto) [reply frontend/src/lib/components/CreateTable.svelte:11]: done

File: frontend/src/lib/components/CreateTable.spec.ts (hunk 2 above)
Comment (Zuretto) [reply frontend/src/lib/components/CreateTable.spec.ts:1]: true. In https://github.com/Zuretto/planning-poker/pull/7 it should be better.

**Pull request closed without merging**